app/controllers/liquid_tags_controller.rb

```diff
@@ -0,0 +1,13 @@
+class LiquidTagsController < ApplicationController
+  before_action :authenticate_user!
+
+  FILTER_REGEX = /^(?:NullTag|Liquid::)/.freeze
+
+  def index
+    custom_tags = Liquid::Template.tags.filter_map do |name, tag|
+      name unless tag.match?(FILTER_REGEX)
+    end.sort
+
+    render json: { liquid_tags: custom_tags }
```

Comment:
@nickytonline I'm not sure about you, but I prefer JSON APIs that always return objects, so I decided to return a response that looks like this:

```json
{
  "liqid_tags": [
    "asciinema",
    "blogcast",
    ...
  ]
}
```